test/e2e/config/azure-dev.yaml

```diff
@@ -195,11 +195,6 @@ variables:
   CLUSTER_IDENTITY_NAME: "cluster-identity-sp"
   NODE_DRAIN_TIMEOUT: "60s"
   CI_VERSION: ""
-  # NOTE: INIT_WITH_BINARY and INIT_WITH_KUBERNETES_VERSION are only used by the clusterctl upgrade test to initialize
-  # the management cluster to be upgraded.
```

Comment:
Are these variables no longer necessary because we're now only using newer versions of CAPI?

Reply:
They were already redundant and not used since we override them with the test spec input variables